### ISSUE TITLE
CDA Header ISO Telecom

### ISSUE THREAD

**Issue by mryanlord** [MDMI/mdmiecore#93]
Telecom elements in USRealmHeader2 annotations not written to .mdmi beyond first element, whether conditions there or not.
occurrences:  Patient; ProviderOrganization; DocumentAuthor; maybe more (pattern established)
Model shows cardinality 0...* for those telecom elements.

**Comment by swmuir** [comment]
Is this still an issue ?  I am able to see the structures etc

**Comment by mryanlord** [comment]
Yes. I still see only one SEER element mapped to telecom in patientRecord and document.author. There should be multiple SEER elements for the multple telecom uses.
